testing/tests/DevExpress.serverSide/__meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "constellation": "misc",
+    "explicit": true,
+    "runOnDevices": true
```

Comment:
Is it necessary to run this tests on devices?

Reply:
No. Fixed.